### PR TITLE
[Add] "Keysmash", a keyboard game

### LINKED ISSUE #2629: Add a Keyboard Game
**Describe the project you want to add with tech stack**
A keyboard game using html, css and javascript. The game is, on displaying the specific key, you need to click on it and it will move to the next key. It is fun to play. Please assign me this project. I would very much like to contribute to it.

**Screenshot**
image

**Additional context:** I am a contributor under GSSOC'23. Please assign this to me under it.

### PR DESCRIPTION
**Pull request by crapbagaah**
# Keyboard game #2629 
I have added a keyboard game
image

It is available both in dark and light mode, just clicking on the mode button at the top right corner changes its mode
image

A keyboard game using html, css and javascript. On displaying the specific key, you need to click on it and it will move on to the next key. It is fun to play.

Please review the PR and add the levels under gssoc'23 as required. Thank you.

Close #2629 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Comment by crapbagaah** [comment]
@MohitGupta121  @TusharKesarwani  @PoulavBhowmick03 please review the PR, thank you

File: Js-Games/Keyboard game/key.jpg
Comment (MohitGupta121) [inline Js-Games/Keyboard game/key.jpg:1]: @crapbagaah  please remove all the images or gif from your PR and use them through cloud link. Upload somewhere the images and use them from that link.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Comment by crapbagaah** [comment]
@MohitGupta121 please review the PR now, I have made the necessary changes

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@crapbagaah images are still present and zip file as well

**Comment by crapbagaah** [comment]
@MohitGupta121 can you review the PR now, I made the necessary changes

**Review by MohitGupta121 — APPROVED** [review]

**Pull request merged**